README.md

```diff
@@ -83,37 +83,47 @@ microovn.ovn-appctl -t /var/snap/microovn/common/run/central/ovnnb_db.ctl cluste
 
 ## Downscaling
 
-> **Warning**
->
-> Be aware of effect of downscaling on availability and resiliency of your
-> cluster, especially when you are removing members that run OVN central
-> service (OVN SB, OVN NB, OVN Northd). OVN uses
-> [Raft consensus algorithm](https://raft.github.io) for cluster management,
-> which has fault tolerance of up to `(N-1)/2` members. It means that if you
-> scale back 3 node cluster to just 2 nodes, you lose any fault resiliency.
+### Impact
 
-Removing MicroOVN cluster member is as easy as running
+Downscaling can have an adverse effect on the availability and resiliency of
+the cluster, especially when a member is being removed that runs an OVN central
+service (OVN SB, OVN NB, OVN Northd).
 
-```shell
-microovn cluster remove <member_name>
-```
+OVN uses the [Raft consensus algorithm](https://raft.github.io) for cluster
+management, which has a fault tolerance of up to `(N-1)/2` members. This means
+that fault resiliency will be lost if a three-node cluster is reduced to two
+nodes.
 
-MicroOVN will stop and disable any chassis components running on the member
-(`ovn-controller` and `ovs-vswitchd`), before attempting to gracefully leave
-any OVN database clusters for members with central components present. You can
-watch logs on departing member for any indications of removal failures with:
+### Monitoring
 
-```shell
-snap logs -f microovn.daemon
-```
+You can watch logs on the departing member for indications of removal failures
+with:
 
-If there are any issues during the removal process, you will have to take
-manual steps to fix the OVN cluster.
+    snap logs -f microovn.daemon
 
-After the removal, you can also check the state of OVN services, to make sure
-that the member was properly removed.
+Any issues that arise during the removal process will need to be resolved
+manually.
 
-```shell
+### Remove a cluster member
+
+To remove a cluster member:
+
+    microovn cluster remove <member_name>
```

Comment:
Is this a preferred way to displaying code blocks?

Reply:
For single or a few lines, yes. Many lines would be bound by triple backticks. At this time, the [Charm Guide](https://docs.openstack.org/charm-guide/latest/community/doc-contrib/md.html#code-blocks) is the reference for OpenStack org formatting.